api/genotypes_loader.api.inc

```diff
@@ -91,6 +109,110 @@ function genotypes_loader_helper_add_record_with_mode($record_type, $table, $mod
   }
 }
 
+/**
+ * Places records into a CSV file, and then checks if it is time to bulk load the file via a COPY command
+ *
+ * @param $record_count
+ *   The current count of records that are being loaded in.
+ * @param $record_type
+ *   A human-readable term for your record for error messages.
+ * @param $table
+ *   The chado table to insert into.
+ * @param $insert_values
+ *   An array of [table column] => [value] mappings that is specific to the insert.
+ * @param $file_name
+ *   An optional name for the file to write the values to
+ * @param $final_chunk
+ *   If set to true, then the remaining genotypes have been added to the file and it is time to copy them in.
+ * @return
+ *   TRUE if success (meaning, we reached the end of the function, not neccesarily that everything went swimmingly), FALSE if failed.
+ */
+
+function genotypes_loader_helper_copy_records($record_count, $record_type, $table, $insert_values = array(), $file_name = NULL, $final_chunk = FALSE) {
+
+  // The number of records we want to reach before copying them in all at once
+  $copy_chunk_size = 10000;
```

Comment:
I would allow this to be set via variable_get/set(). 
1. Change this line to `$copy_chunk_size = variable_get('genotypes_loader_cp_chunk_size', 10000)` 
2. Set via variable_set() in the first drush function after exposing an option to the user.
This way you don't have to worry about trying to pass the value through the chain of functions while still providing a way for your user to change this value per file.

Reply:
I created issue #20 to address this!